dashboard/src2/components/settings/profile/TFARecoveryCodes.vue

```diff
@@ -0,0 +1,72 @@
+<template>
+	<div class="space-y-4">
+		<AlertBanner
+			title="Save these recovery codes in a safe place. You can use them to
+				reset two factor authentication if you lose access to your
+				authenticator app."
+			type="warning"
+		/>
+
+		<div class="rounded border border-gray-200 bg-gray-50 p-4">
+			<div class="font-mono text-sm leading-loose text-gray-700">
+				{{ recoveryCodesStr }}
+			</div>
+		</div>
```

Comment:
optional but use `ClickToCopyField` component here (users will be able to copy the codes)